src/components/Molecules/Footer/Footer.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Link from '../../Atoms/Link/Link';
+
+export const Inner = styled.footer`
+  background-color: black;
+
+  ul li a {
+    color: white;
```

Comment:
Here you can do something like:
```
const NavLink = styled(Link)`
 color: ${theme => theme.color('white');
`;
```
You can style our Link and use our theme colours.